docs/source/index.rst

```diff
@@ -3,10 +3,13 @@ The Jupyter notebook
 ====================
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
    :caption: User Documentation
 
    notebook
+   Installation <https://jupyter.readthedocs.org/en/latest/install.html>
```

Comment:
@minrk Unfortunately, a limitation of Sphinx is that intersphinx links are not supported in toctrees.